### DISCUSSION TITLE
Keycloak.X k8s getting started

### DISCUSSION THREAD

**Discussion by bjolivot** [keycloak/keycloak#9154] [Q&A] [answered]
Hello !
Wath's the best way to start with Keycloak.X on kubernetes (here on GKE) ?
Is the Operator ready ?
Do I need to build custom manifests ?

Is there some recomanded SQL DB ? 

Thank you !

**Comment by vmuzikar** [comment, marked as answer]
Hello,
the best way to use Keycloak.X in K8s will be the new operator. We actually published our [design proposal](https://github.com/keycloak/keycloak-community/pull/330/files) today, so feel free to check it out. Feedback would be much appreciated!

In the meantime it is of course possible to deploy and configure the [Keycloak.X image](https://github.com/keycloak/keycloak-containers/tree/main/server-x) directly.

**Reply by bjolivot** [reply]
thank you! 
that's the doc I was looking for :)

...few hours later : 2021-12-16 19:18:36,737 INFO  [io.quarkus] (main) Keycloak 15.1.0 on JVM (powered by Quarkus 2.5.1.Final) started in 8.784s. Listening on: http://0.0.0.0:8080
Yeah!
